test/test_lswt.jl

```diff
@@ -60,7 +60,10 @@
     # to one arbitrary k-vector, and round to 10 digits of accuracy.
     formula = intensity_formula(swt,:full,kernel = delta_function_kernel,formfactors = [FormFactor("Fe2")])
     _, is_full = intensities_bands(swt,[ks[1]],formula)
-    is_full_flattened = reinterpret(reshape, ComplexF64, is_full)[:]
+    is_full_flattened = zeros(ComplexF64,720)
+    for i = 1:80
+        is_full_flattened[9*(i-1) .+ (1:9)] .= is_full[i][:]
+    end
```

Comment:
Why are these additional lines needed?

Reply:
Previously, this read `is_full_flattened = reinterpret(reshape, ComplexF64, is_full)[:]`, which fails because `SMatrix{3,3,ComplexF64}` isn't `isbits`. I'm not sure how this ever worked in the first place, but this code simply converts the intensities produced by the code to the shape of the golden reference intensities.

Perhaps a `Matrix` should be returned instead of `SMatrix`? Would that improve things?

Reply:
That's strange, I see this at the terminal:
```julia
m = randn(Sunny.Mat3)
@assert isbits(m)
```
I will think about it tomorrow and see if there's some reinterpret/reshape that's possible.

Reply:
I agree there's something strange here. Let's see what they say here: https://github.com/JuliaArrays/StaticArrays.jl/issues/1201